config/routes.rb

```diff
@@ -606,7 +606,7 @@ def call(env)
         resources :features, :controller => 'account_plan_features', :only => [:index, :create, :destroy]
       end
 
-      resources :active_docs, :controller => 'api_docs_services', :only => [:index, :update, :create, :destroy]
+      resources :active_docs, :controller => 'api_docs_services', :only => %i[index show update create destroy]
```

Comment:
All the API routes 😄 Maybe shorter:
```suggestion
      resources :active_docs, :controller => 'api_docs_services', except: %i[new edit]
```

Either way, it is good 👍

Reply:
Done